worker/src/RTC/Codecs/BitStream.cpp

```diff
@@ -0,0 +1,52 @@
+#define MS_CLASS "RTC::Codecs::BitStream"
+
+#include "RTC/Codecs/BitStream.hpp"
+#include "Logger.hpp"
+
+namespace RTC
+{
+	namespace Codecs
+	{
+		BitStream::BitStream(const uint8_t* data, size_t len) : data(data), len(len)
```

Comment:
Does it make sense that this class is inside Codecs? We may need it to parse whatever.

Reply:
Yep, I was definitely thinking of moving it out. Maybe under `RTC` directly?

Reply:
I clearly see it in Utils namespace which is outside RTC.